src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/CustomiseHearingBundleConfirmationTest.java

```diff
@@ -1,16 +1,22 @@
 package uk.gov.hmcts.reform.iacaseapi.domain.handlers.postsubmit;
 
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.hamcrest.Matchers.containsString;
-import static org.junit.Assert.*;
+import static org.hibernate.validator.internal.util.Contracts.assertNotNull;
```

Comment:
Though it works I think we do not need the hibernate library for this. Could you please use assertj library instead to be in consistent.

Reply:
Corrected Assert statements